### ISSUE TITLE
EPUB 2 ONIX missing translation of a11y:certifiedBy

### ISSUE THREAD

**Issue by LaZay** [w3c/a11y-discov-vocab#56]
The EPUB `a11y:certifiedBy `metadata has an equivalent in ONIX: code list [196](https://ns.editeur.org/onix/en/196), code [93](https://ns.editeur.org/onix/en/196/93) (_Compliance certification by_).
Could you correct this please? As a matter of fact, the lack of translation is very misleading on ONIX expressive power, for those who are not ONIX experts.
